### PR TITLE
fix lack of reserveFace

### PR DESCRIPTION
**Pull request by krmbn0576**
Hey, I fixed a lack of some reserveFaces.
For `Window_Status and Window_SkillStatus`, a face image is necessary, but it was not reserved.
This is a bug from the original code, but now is a good opportunity to fix it.
So, I reserved these face images when each scene (`Scene_Status, Scene_Skill`) started.

This is a breaking change because "Now Loading" can occur when the scene starts.
However, in fact it will not happen, because the cache mechanism is LFU,
so the face image will be loaded by `Window_MenuStatus` just before.

### CODE REVIEW

**Comment by krmbn0576** [comment]
I added a reservation for `Window_Message` for a different reason than above.
This is a modification of the en-bug accompanying the implementation change of `ImageManager.isReady()`.
Certainly a face image isn't purged until it's ready.
However, it can be purged after it's ready. See below.

Face --------> Ready! ------> Purged!?
Other -----------------------------------------------------> Ready! (Imagemanager.isReady() === true)

It is hesitant to further complicate remodeling `ImageManager.isReady()` any more.
Therefore I simply reserved the face image.

**Comment by ghost** [comment]
> This is a modification of the en-bug accompanying the implementation change of ImageManager.isReady().

It's not `isReady()`'s problem, just there are chance to purge bitmap.
but solution is correct. I write `reserveXXX` for this purpose.

**Comment by ghost** [comment]
If you think about cohesion of Windows, This solution is not valid.

ex: Window_SkillStatus
reserves face and uses it directly. consider the case face images are not ready.

We cannot change signature of public method, so this is why I say it doesn't need.

**Comment by krmbn0576** [comment]
OK, I made it load all members face images when scene starts.

File: js/rpg_scenes/Scene_Status.js [hunk 1]
```diff
@@ -24,6 +24,11 @@ Scene_Status.prototype.create = function() {
     this.refreshActor();
```
Comment (ghost) [inline js/rpg_scenes/Scene_Status.js]: This can cause performance problem.

File: js/rpg_scenes/Scene_Status.js [hunk 2]
```diff
@@ -24,6 +24,11 @@ Scene_Status.prototype.create = function() {
...
 };
 
+Scene_Status.prototype.start = function() {
+    Scene_MenuBase.prototype.start.call(this);
+    this._statusWindow.refresh();
```
Comment (ghost) [inline js/rpg_scenes/Scene_Status.js]: Why not remove above and change to `refreshActor()`?

File: js/rpg_scenes/Scene_Skill.js [hunk 3]
```diff
@@ -24,6 +24,11 @@ Scene_Skill.prototype.create = function() {
     this.refreshActor();
 };
 
+Scene_Skill.prototype.start = function() {
+    Scene_ItemBase.prototype.start.call(this);
+    this._statusWindow.refresh();
```
Comment (ghost) [inline js/rpg_scenes/Scene_Skill.js]: same below.

File: js/rpg_scenes/Scene_Status.js (hunk 1 above)
Comment (krmbn0576) [reply js/rpg_scenes/Scene_Status.js]: Fixed!

**Comment by ghost** [comment]
Reserve and release timing is not valid if you think about cohesion.
those are caller's responsibility, 

And linking all face images to one reservationId causes cohesion problem.

**Comment by krmbn0576** [comment]
Hm, I moved call `reserveFace` timing to `Scene_Xxx`.

**Review by ghost — APPROVED** [review]

**Pull request merged**